src/collar/oc_exceptions.lsl

```diff
@@ -226,62 +226,72 @@ SaveSettings() {
 
 SetAllExs() {
     if (!g_iRLVOn) return;
+    ClearEx(); //Previously this script sets every exception to y or n. It only needs to set y if the exception is already set to n. Either we a: spam the viewer with unset commands as current, b: get current exceptions and parse the list to remove / set as needed, or c: clear exceptions and start from scratch, which is frankly a lot simpler.
     integer iStop = llGetListLength(g_lRLVcmds);
     integer n;
     integer i;
-    string sRLVCmd = "@";
+    //string sRLVCmd = "@";
+    list lRLVCmd; // Combined RLV commands need comma separation, so let's do it with a list for ease. It's lucky the llOwnerSay commands were being accidentally issued inside the loop for owner/secowner as commas weren't being added, which breaks things. So... yay, a bug that fixed another bug! Let's fix both though.
     integer iLength = llGetListLength(g_lSecOwners);
     for (n = 0; n < iLength; ++n) {
         string sTmpOwner = llList2String(g_lSecOwners, n);
-        if (llListFindList(g_lSettings, [sTmpOwner]) == -1 && sTmpOwner!=g_kWearer) {
-            for (i = 0; i<iStop; i++) {
+        if (llListFindList(g_lSettings, [sTmpOwner]) == -1 && sTmpOwner!=g_kWearer) 
```

Comment:
It's not a blocker, but I have a small preference for curly braces going on the same line.  It's semi important to keep it consistent across the project, and more important to keep it consistent within the same script.  Unless you're going to make a pass through the whole project to put them all on separate lines, I'd prefer to keep the style used elsewhere.

As a bonus, curly-brace-on-same-line  in other languages (javascript) can prevent subtle bugs.  https://stackoverflow.com/questions/3218756/javascript-braces-on-new-line-or-not (See the second answer.)